apps/ewallet_db/lib/ewallet_db/expirers/auth_expirer.ex

```diff
@@ -27,10 +27,12 @@ defmodule EWalletDB.Expirers.AuthExpirer do
   @spec get_advanced_datetime(integer) :: NaiveDateTime.t() | nil
   def get_advanced_datetime(0), do: nil
 
-  def get_advanced_datetime(second) do
+  def get_advanced_datetime(second) when is_integer(second) and not is_nil(second) do
```

Comment:
```suggestion
  def get_advanced_datetime(second) when is_integer(second) do
```

Remove redundant guard, `is_integer/1` should already cover `not is_nil(..)`.